B827EBFFFE0F6CF6.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE0F6CF6",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 51.459305,
+    "ref_longitude": 7.013327,
+    "ref_altitude": 40,
+    "contact_email": nils@tekampe.org",
```

Comment:
There's a missing starting quote here, which makes the file invalid json.

Reply:
just fixed it